.devcontainer/Dockerfile

```diff
@@ -8,6 +8,6 @@ RUN install-apt make g++
 RUN install-tool python 3.11.4
 
```

Comment:
I still see some references to `yarn` in the code:

- [ ] `.prettierignore` file
- [ ] `"keywords"` array in `package.json` file
- [ ] `yarn test` reference in `.github/contributing.md`
- [ ] `yarn.lock` reference in `.github/actions/calculate-prefetch-matrix`
- [ ] `"runtimeExecutable": "yarn",` in `.vscode` file
- [ ] `"jest.jestCommandLine": "yarn jest"`, in `settings.json`
- [ ] `"npm.packageManager": "yarn"`, in `settings.json`

Reply:
`keywords` is ok, as it's more for renovate functionallity :hugs